### ISSUE TITLE
[valid-title] static class string name property not allowed

### ISSUE THREAD

**Issue by olivierlacan** [jest-community/eslint-plugin-jest#1390]
Given: 

```typescript
export class SomethingModule {}
```

```typescript
describe(SomethingModule.name, function () {}
```

In 27.2.2, we get this error: 
> Title must be a stringeslint[jest/valid-title](https://github.com/jest-community/eslint-plugin-jest/blob/v27.2.2/docs/rules/valid-title.md)

The static read-only name of a class should be allowed here, it's typed as a string. While it's dynamic, it seems perfectly fine as a `describe` name when making a unit test.

**Comment by G-Rath** [comment]
That's because we can't know if something is a class or not, because ESLint doesn't have that kind of type information available.
